news/2025/2025-01-27-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,210 @@
+- [Yousei Teikoku - Zetsubou plantation (Cut. Ver) [PikA's Extra]](https://osu.ppy.sh/beatmapsets/2250663#osu/4793476) with DT: 10.46* --> 10.22*
+
+As part of this change, the wiggle bonus which used to be part of the acute angle bonus is now applied regardless of BPM - beatmaps with difficult wiggle-like patterns below 300BPM may receive a (slight) new buff!
+
+### Slider drop penalty fixes
+
+Aim calculations reward beatmaps for having difficult to follow slider patterns, with a penalty for dropped slider ends to ensure that plays only receive their full buff for these sliders if they were hit correctly.
+
+A [change](https://github.com/ppy/osu/pull/31055) proposed by [StanR](https://osu.ppy.sh/users/7217455) has been created in order to ensure this penalty is applied correctly, and uses a more concrete estimation of the difficult sliders in a beatmap.
+
+Prior to this change, the penalty applied to aim PP for missed sliderends was unintentionally nerfing much less than it was supposed to. This meant beatmaps with difficult sliders could be abused for easy PP by missing a lot of slider ends. This has been corrected which causes large nerfs for a lot of scores set on beatmaps such as [t+pazolite - Oshama Scramble! (IOException Edit) [Special]](https://osu.ppy.sh/beatmapsets/1376308#osu/2844649).
+
+Furthermore, this change now uses a count of difficult sliders dictated by difficulty calculations for this penalty instead of assuming that 10% of the beatmap's sliders are difficult. This makes the penalty fairer across the board.
+
+### Speed PP punishment for scores with high tapping deviation
+
+A [change](https://github.com/ppy/osu/pull/30907) proposed by [Givikap120](https://osu.ppy.sh/users/10560705) has been created in order to nerf speed PP on scores with a very high tapping deviation.
+
+Tapping deviation is the estimated result (similar to unstable rate) of a statistical calculation created by [Frostium](https://osu.ppy.sh/users/8202998) using the score's hit statistics, OD and amount of difficult speed notes in the beatmap.
+
+This deviation is used to punish speed PP if the deviation is deemed "too high" for the beatmap's difficulty and is assumed to have been tapped improperly. This primarily addresses scores using the "rake tapping" technique while also nerfing other forms of improper tapping.
+
+This nerf works by deciding a cut-off point for speed PP, scaled by the deviation, where anything above it is considered to be tapped improperly - scores which go above this cut-off point are then re-adjusted based on their deviation to give a more representative value of their tapping difficulty.
+
+The nerf has a threshold for deviation that must be reached before it nerfs anything at all in order to ensure the majority of scores which were tapped correctly are either not nerfed or nerfed insignificantly.
```

Comment:
this paragraph doesn't seem to add anything. wasn't the cut-off/threshold just explained above?

```suggestion
```

Reply:
Perhaps explained poorly, the cut-off and threshold are 2 different things. If your PP is already below the cut-off then no nerf applies. If your PP is above the cut-off but your deviation is below the threshold then no nerf applies. If your PP is above the cut-off **and** your deviation is above the threshold, then the nerf applies.

Reply:
is the "cut-off" always below the "threshold"? that'd make it truly useless

and if not, then they're named weirdly. you just have two cut-offs/thresholds, or rather, can't we just say they're the same thing if they do the same thing?

if there's some more complication that warrants this then i guess i'll just read the code

Reply:
The cut-off and threshold are measured against 2 different things:

cut-off - speed PP
threshold - deviation

i.e they're not "below" or "above" each other, they're not to be compared

Reply:
Let me try to explain this again, because I think the issue is my explanation is poor.

The first step of the nerf uses your speed deviation to create a speed PP value that acts as a "cut off" - if your speed PP is below this cut off point, it exits out and no nerf is applied.

The second step of the nerf then looks at your speed deviation. If your speed deviation is below the threshold for any nerf to apply, it exits out and no nerf is applied.

If you reach the third step, your speed PP is above the cut-off point and your deviation is above the threshold. In this case, your deviation is then used alongside the cut-off point to scale down the speed PP of your score.